### PR TITLE
Fix import error on stock_inventory_hierarchical_location

### PR DESCRIPTION
**Pull request by andhit-r**
Fix issue https://github.com/OCA/stock-logistics-warehouse/issues/152

### CODE REVIEW

File: stock_inventory_hierarchical_location/inventory_hierarchical_location.py [hunk 1]
```diff
@@ -21,10 +21,10 @@
 from openerp.osv import orm
 from openerp.tools.translate import _
 
-from stock_inventory_hierarchical import HierarchicalInventoryException
+from openerp.addons.stock_inventory_hierarchical.exceptions import HierarchicalInventoryException
```
Comment (pedrobaeza) [inline stock_inventory_hierarchical_location/inventory_hierarchical_location.py]: The more suitable form is `from .stock_inventory_hierarchical import ...`

**Comment by andhit-r** [comment]
@pedrobaeza done

**Comment by pedrobaeza** [comment]
Sorry, I though the file was in the same module, but it isn't. The way you put it was correct. Sorry for making you change it.

**Comment by andhit-r** [comment]
@pedrobaeza no worries. I didn't check it either. Still in panic mode because all my servers went down after pulled latest code :)

**Comment by pedrobaeza** [comment]
Let's fast-track this one then.

:+1:

**Comment by Daniel-CA** [comment]
I am having another error in this module now.
  File "/opt/odoo/demo_odoomrp/oca/stock-logistics-warehouse/stock_inventory_hierarchical_location/inventory_hierarchical_location.py", line 24, in <module>
    from openerp.addons.stock_inventory_hierarchical.exceptions import HierarchicalInventoryException
ImportError: No module named exceptions

**Comment by Daniel-CA** [comment]
Forget about this, looks like that there is some kind of problem in the dependencies of the linux server, I am having another problem in another module so probably fixing dependency problem will fix this too.

**Pull request merged**